### PR TITLE
Add TM_SUBTASK_NAME to the environment for validators.

### PR DESCRIPTION
**Pull request by veluca93**

### CODE REVIEW

File: task-maker-format/src/ioi/dag/input_validator.rs [hunk 1]
```diff
@@ -36,6 +36,7 @@ impl InputValidator {
         eval: &mut EvaluationData,
         description: String,
         subtask_id: SubtaskId,
+        subtask_name: &Option<String>,
```
Comment (edomora97) [inline task-maker-format/src/ioi/dag/input_validator.rs:39]: What about `Option<&str>`? This way you can pass `None` or `subtask.name.as_ref()`.

File: task-maker-format/src/ioi/sanity_checks/att.rs [hunk 2]
```diff
@@ -128,6 +128,7 @@ impl SanityCheck<IOITask> for AttSampleFilesValid {
                     eval,
                     format!("Validation of sample case {}", input_name.display()),
                     0,
+                    None,
```
Comment (edomora97) [inline task-maker-format/src/ioi/sanity_checks/att.rs:131]: I'm wondering if this should be the name of the subtask with the examples. Not sure how to reliably get that, or whether a new subtask name should be used (i.e. "att"). Probably leaving that empty is fine for now.

File: task-maker-format/src/ioi/dag/mod.rs [hunk 3]
```diff
@@ -229,7 +229,7 @@ mod tests {
         let file = File::new("input");
         let (mut eval, _recv) = EvaluationData::new(tmpdir.path());
         let out = validator
-            .validate_and_bind(&mut eval, 0, 0, file.uuid)
+            .validate_and_bind(&mut eval, 0, None, 0, file.uuid)
```
Comment (edomora97) [inline task-maker-format/src/ioi/dag/mod.rs:232]: I'd appreciate also a test where this parameter is used, together with a test where it's None.

**Review by edomora97 — CHANGES_REQUESTED** [review]
Apart from the comment about the test, it looks fine! Thanks!

**Comment by edomora97** [comment]
Oh, and after adding the test you can rebase everything 👍🏻

**Review by edomora97 — APPROVED** [review]
